openlibrary/macros/FulltextSearchSuggestion.html

```diff
@@ -0,0 +1,35 @@
+$def with(query, results, page=1)
+
+$def render_snippet(query, doc):
+    $:macros.FulltextSuggestionSnippet(query, doc=doc)
+
+$if results and results.get('hits'):
+  $ hits = results['hits'].get('hits', [])[:4]
+  $ num_found = commify(results['hits'].get('total', 0))
+  $ ia_base_url = "https://archive.org"
```

Comment:
Is this used anywhere in this file?
```suggestion
```

Reply:
@mekarpeles It's not ! I will take this out and repush